docs/snowflake-native-apps/facebook-marketing.md

```diff
@@ -0,0 +1,207 @@
+# Facebook Marketing Connector
+
+The Facebook Marketing Connector by Airbyte is a Snowflake Native Application that allows you to extract data from your Facebook Marketing account and load records into a Snowflake database of your choice.
+
+:::info
+The Snowflake Native Apps platform is new and rapidly evolving. The Facebook Marketing Connector by Airbyte is in _private preview_ and is subject to further development that may affect setup and configuration of the application. Please note that, at this time, only a full table refresh without deduplication is supported.  
```

Comment:
Probably makes sense to link to the Airbyte full refresh documentation here.

Reply:
will do!